Rules/UseConsistentIndentation.cs

```diff
@@ -255,6 +257,21 @@ public override IEnumerable<DiagnosticRecord> AnalyzeScript(Ast ast, string file
             return diagnosticRecords;
         }
 
+        private static CommandBaseAst LastPipeOnFirstLineWithPipeUsage(PipelineAst pipelineAst)
+        {
+            CommandBaseAst lastPipeOnFirstLineWithPipeUsage = pipelineAst.PipelineElements[0];
+            foreach (CommandBaseAst pipelineElement in pipelineAst.PipelineElements.Skip(1))
```

Comment:
It feels like a regular `for` loop would be more natural here, but just a suggestion